include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -650,6 +650,66 @@ struct OrtApi {
   ORT_CLASS_RELEASE(TensorTypeAndShapeInfo);
   ORT_CLASS_RELEASE(SessionOptions);
   ORT_CLASS_RELEASE(CustomOpDomain);
+
+  // End of Version 1 - DO NOT MODIFY ABOVE (see above text for more information)
+
+  // Version 2 - In development, feel free to add/remove/rearrange here
+
+  /**
+    * GetDenotationFromTypeInfo
+	 * This api augments OrtTypeInfo to return denotations on the type.
+	 * This is used by WinML to determine if an input/output is intended to be an Image or a Tensor.
+    */
+  OrtStatus*(ORT_API_CALL* GetDenotationFromTypeInfo)(_In_ const OrtTypeInfo*, _Out_ const char** const denotation, _Out_ size_t* len)NO_EXCEPTION;
```

Comment:
[nit] Add some spaces for readability before and after the function? Or is stupid clang-format messing things up?

`OrtStatus* (...)(...) NO_EXCEPTION;`

Reply:
Ah, seems you just cut&paste this from elsewhere. Non-blocking minor.

Reply:
same formatting as other places in this file!

---
In reply to: [374407768](https://github.com/microsoft/onnxruntime/pull/2970#discussion_r374407768) [](ancestors = 374407768)

Reply:
yep

---
In reply to: [374408058](https://github.com/microsoft/onnxruntime/pull/2970#discussion_r374408058) [](ancestors = 374408058)